Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:latest
+
+RUN apt-get update
+RUN apt-get install -y python3-dev python3 python3-pip curl unzip git jq
+
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && \ 
+    unzip awscliv2.zip && \
+    ./aws/install
+
+RUN update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+
+ADD ./requirements.txt ./
+ADD ./tests ./tests
+
+RUN aws --version
```

Comment:
Why do we need this command?

Reply:
It's nice to see the aws version running in the logs

Reply:
Thanks, was left over from testing